api/optimeet/optimeet/settings.py

```diff
@@ -95,6 +95,8 @@
 }
 
 
+
+
```

Comment:
Be sure to double check the changes you've made before committing through the Source Control tab in VS Code!